src/libraries/prisma/prisma.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable, type OnModuleInit } from '@nestjs/common';
+import { PrismaClient } from '@prisma/client';
+
+// @Injectable() allows NestJS to manage this service's lifecycle
+// and inject it as a dependency into other services/controllers
+@Injectable()
```

Comment:
Dependency injection  💉  so nest can handle the singleton pattern for us, among other goodies